### DISCUSSION TITLE
[admin-boundaries]

### DISCUSSION THREAD

**Discussion by waelhabaieb** [OvertureMaps/data#181] [Discussion]
### Category Feedback

Add details 
 - Expected behaviour you didn't see 
 - Unexpected behaviour 
 - suggested functionality 
 - Steps to reproduce the problem

**Comment by jwass** [comment]
Hi @waelhabaieb Looks like you might have forgotten to fill this out with more info. I'll close it but feel free to update and re-open or start a new discussion.
